README-CONAN.md

```diff
@@ -257,6 +255,27 @@ $ cmake ..
 CMake should manage to configure the project finding all the dependencies. If you want to know more about how to use
 CMake in this project, refer to [README-CMAKE](README-CMAKE).
 
+### Linux specific notes
+
+One of the most important settings to be adjusted in your conan profile when working on Linux is the field
+
+```bash
+compiler.libcxx=libstdc++11  # Possible values: libstdc++, libstdc++11, libc++
+```
+With the arrival of the c++11 standard, and the popularity obtained by the *clang* compiler in the last years, it became more and more important to choose at configuration time which version of the standard library to use (with the `-stdlib` compiler flag).
+
+In this link about the [dual ABI in gcc](https://gcc.gnu.org/onlinedocs/libstdc++/manual/using_dual_abi.html), you will find some interesting information about how the standard library released with GCC works. This is how the **libcxx** settings works with conan and different compilers:
+
+```bash
+compiler.libcxx=libstdc++11  # will use -stdlib=libstdc++ and define _GLIBCXX_USE_CXX11_ABI=1
+compiler.libcxx=libstdc++    # will use -stdlib=libstdc++ and define _GLIBCXX_USE_CXX11_ABI=0
+compiler.libcxx=libc++       # will use -stdlib=libc++
+
+```
+Please, be aware that normally when using gcc >= 5.1, _GLIBCXX_USE_CXX11_ABI is set to 1 by default. However some linux distributions might set that definition to 0 by default. In case you get linking errors about standard c++ containers or algorithms when bringing the Exiv2 dependencies with conan, this might indicate a mismatch between the value set in **compiler.libcxx** and the default values used in your distribution.
```

Comment:
As far as I know no distro did that.

Reply:
I took that sentence from the GCC link about the Dual ABI:

```
(Be aware that some GNU/Linux distributions configure GCC 5 differently so that the default value of the macro is 0 and users must define it to 1 to enable the new ABI.)
```
I only use Ubuntu regularly and I do not know if this happens in other popular distributions. However I noticed that some conan recipes set `_GLIBCXX_USE_CXX11_ABI` with different values depending on the **compiler.libcxx** to deal with this kind of issues. One example is available in one of the most popular conan recipes: conan-boost:

https://github.com/conan-community/conan-boost/blob/release/1.66.0/conanfile.py